gems/aws-sdk-core/spec/fixtures/waiters/service.json

```diff
@@ -27,7 +27,13 @@
       "type":"structure",
       "members":{
         "Table":{"shape":"TableDescription"},
-        "TableList":{"shape":"TableList"}
```

Comment:
This example was pretty much retrofitted by dynamodb. If it's not too much trouble, we should probably slightly rewrite this - maybe we can do away with the "Table" concepts, make the operation just called operation, etc. Isn't TableList just the same as ObjectList pretty much?